pkg/occlient/occlient.go

```diff
@@ -1197,18 +1208,28 @@ func (c *Client) BootstrapSupervisoredS2I(params CreateArgs, commonObjectMeta me
 	jsonDC, _ = json.Marshal(createdDC)
 	glog.V(5).Infof("Created new DeploymentConfig:\n%s\n", string(jsonDC))
 
-	svc, err := c.CreateService(commonObjectMeta, dc.Spec.Template.Spec.Containers[0].Ports)
+	// we create a ownerReference struct so that we can set the DC created
+	// above as owner of the ImageStream created earlier, and Service, Secret
+	// to be created next.
+	ownerReference := metav1.OwnerReference{
```

Comment:
We have the same ownersReference definition on multiple places, we should abstract this away to have it defined only once.